Yoti.Auth.Sandbox/DocScan/Request/Resource/Document/DocumentResource.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+
+namespace Yoti.Auth.Sandbox.DocScan.Request.Resource.Document
+{
+    public class DocumentResource : SandboxPayload
```

Comment:
This should live with the integration tests (web client)

Reply:
Moved in ddbb848